doc/cheatsheet_perl.md

```diff
@@ -86,6 +123,11 @@ If you are unfamiliar with these functions, they are explained more extensively
 * `$<v> =~ s/regex//` -- substitution
 * `$<v> = tr/regex//` -- transliteration (also can be done with `y/regex//`)
 
+### Logical and Existence Operators
+* `exists` -- truthy when a key exists in a hash, falsey otherwise
+* `defined` -- truthy when a value is not `undef`, falsey otherwise
+* `&&`, `||` -- high priority boolean operators; use to perform boolean operations directly on scalar values 
```

Comment:
this might be a good place to mention `//`; only the oldest perls don't support it